### DISCUSSION TITLE
Can't send messages in Copilot

### DISCUSSION THREAD

**Discussion by danilohrf** [community/community#147796] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm trying to use Copilot in VSCode through the chat or edit, but I can't seem to be able to send a message. I've read old threads about switching to pre-release version, login out and login back in, restarted VSCode, restarted my PC and nothing works. Can anyone help?

**Comment by danilohrf** [comment, marked as answer]
This time uninstalling and reinstalling worked. Sorry.

**Reply by gtoto007** [reply]
I had the issue, and I resolved it by reinstalling it
